<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP-Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="remotePortLabel">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Удаленный порт:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Отправить</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="localPortLabel">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Локальный порт : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QLineEdit" name="messageLine"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="localPortLine"/>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Подключиться</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="disconnectButton">
        <property name="text">
         <string>Отключиться</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="remotePortLine"/>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QTextBrowser" name="messageTextBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectClicked()</slot>
  <slot>sendClicked()</slot>
  <slot>disconnectClicked()</slot>
 </slots>
</ui>
